client/src/views/Exchange.tsx

```diff
@@ -0,0 +1,245 @@
+import { useEffect, useState } from 'react'
+import { Block } from '../components/shared/Block'
+import Box from '@mui/material/Box'
+import Chip from '@mui/material/Chip'
+import { periods } from '../constants'
+import { ExchangeService, MailBoxUsageDetail } from '../services/exchange'
+import { formatBytestToGB } from '../utils/helpers'
+import { ExchangeList } from '../components/ExchangeList'
+import { PeriodValueInDays } from '../types/general'
+import { columnDefExchanges } from '../columnsDef/exchange'
+import { BoxLoader } from '../components/shared/Loaders/BoxLoader'
+
+interface TypeOfStatsData {
+    totalMailboxesCount: string
+    activeMailboxesCount: string
+    inactiveMailboxesCount: string
+    activeVersusTotalMailboxes: string
+    totalStorage: string
+    sharedMailboxesCount: string
+    resourceMailboxesCount: string
+    totalContactsCount: string
+    emailReadsCount: string
+    emailReceivedCount: string
+    emailSentCount: string
+    activeUsers: string
+    outlookMobile: string
+    outlookWindows: string
+    outlookWeb: string
+    outlookOther: string
+}
+
+export const Exchange = () => {
+    const [selectedPeriod, setSelectedPeriod] = useState<PeriodValueInDays>(30)
+
+    const [isLoading, setIsLoading] = useState(true)
+
+    const [activityDetails, setActivityDetails] = useState<MailBoxUsageDetail[]>([])
+
+    const [statsDataFromApi, setStatsDataFromApi] = useState<TypeOfStatsData>({
+        totalMailboxesCount: '',
+        activeMailboxesCount: '',
+        inactiveMailboxesCount: '',
+        activeVersusTotalMailboxes: '',
+        totalStorage: '',
+        sharedMailboxesCount: '',
+        resourceMailboxesCount: '',
+        totalContactsCount: '',
+        emailReadsCount: '',
+        emailReceivedCount: '',
+        emailSentCount: '',
+        activeUsers: '',
+        outlookMobile: '',
+        outlookWindows: '',
+        outlookWeb: '',
+        outlookOther: '',
```

Comment:
Minor: The idea of this to be a class, so that you can use new TypeOfStatsData() rather than all this empty stirngs,

Reply:
fixed